### PR TITLE
feature(main): buildx user-controller

### PR DESCRIPTION
**Pull request by cuisongliu**
Signed-off-by: cuisongliu <cuisongliu@qq.com>

### CODE REVIEW

**Comment by zzjin** [comment]
Maybe it's a better workflow structure usage:

put real actions at `./controllers/user/.github/workflows/ci.yml` like 
```yaml
name: CI

on:
  workflow_dispatch:
  push:
    branches: [ "main" ]
    paths-ignore:
      - "docs/**"
      - "**/*.md"
      - "**/*.yaml"
  pull_request:
    branches: [ "*" ]
    paths-ignore:
      - "docs/**"
      - "**/*.md"
      - "**/*.yaml"
```
, and use `uses` at `.github/workflows/controllers.yml`:
```yaml
jobs:
  controllers:
    steps:
      - name: User Controller
        uses: ./controllers/user/.github/workflows/ci
```

**Comment by cuisongliu** [comment]
> Maybe it's a better workflow structure usage:
> 
> put real actions at `./controllers/user/.github/workflows/ci.yml` like
> 
> ```yaml
> name: CI
> 
> on:
>   workflow_dispatch:
>   push:
>     branches: [ "main" ]
>     paths-ignore:
>       - "docs/**"
>       - "**/*.md"
>       - "**/*.yaml"
>   pull_request:
>     branches: [ "*" ]
>     paths-ignore:
>       - "docs/**"
>       - "**/*.md"
>       - "**/*.yaml"
> ```
> 
> , and use `uses` at `.github/workflows/controllers.yml`:
> 
> ```yaml
> jobs:
>   controllers:
>     steps:
>       - name: User Controller
>         uses: ./controllers/user/.github/workflows/ci
> ```
> 
>  > https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-an-action-in-the-same-repository-as-the-workflow

Maintain the status quo first and then optimize it later.

File: service/payment/api/api.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ import (
 	"os"
 	"strconv"
 
-	"github.com/labring/sealos/pkg/pay"
```
Comment (zzjin) [inline service/payment/api/api.go:24]: services must not require other `services`/`controllers`

Comment (cuisongliu) [reply service/payment/api/api.go:24]: Is there no solution to this problem?

Dockerfile doesn't seem to be able to introduce sealos/pkg

**Review by zzjin — APPROVED** [review]

**Pull request merged**